### ISSUE TITLE
About the Batchsize

### ISSUE THREAD

**Issue by lixingang** [sadeepj/crfasrnn_keras#48]
I notice the README showing the batchsize must equal one, 
But I write a simple training code about classification problem ,  just applying Keras API with batchsize==5. 
`model.fit(data, labels, epochs=10, batch_size=5)`
And there is no error when running it. May there be some latent incorrectness?

**Comment by lixingang** [comment]
Thx
